### PR TITLE
Improved Blackhole entity

### PR DESCRIPTION
**Pull request by TheOnly8Z**
Behavior for func_physbox and func_breakable.

### CODE REVIEW

File: lua/entities/ent_jack_gmod_ezblackhole.lua [hunk 1]
```diff
@@ -6,8 +6,10 @@ ENT.Category="JMod - EZ"
 ENT.Information="glhfggwpezpznore"
 ENT.PrintName="EZ Micro Black Hole"
 ENT.Spawnable=false
-ENT.AdminSpawnable=false
+ENT.AdminSpawnable=true
```
Comment (Jackarunda) [inline lua/entities/ent_jack_gmod_ezblackhole.lua]: i don't want the black hole to show up in the spawn menu in any case, it's a point entity

File: lua/entities/ent_jack_gmod_ezblackhole.lua [hunk 2]
```diff
@@ -29,7 +31,13 @@ function ENT:SUCC(Time,Phys,Age,Pos,MaxRange)
 			end
 		elseif(table.HasValue(self.RagdollifyEnts,Class))then
 			if((SERVER)and(math.random(1,100)==42))then obj:Fire("becomeragdoll","",0) end
-		elseif((IsValid(ObjPhys))and not(obj:IsWorld())and not(obj==self)and not(self:IsBlacklisted(obj)))then
+		elseif table.HasValue(self.DamageEnts, Class) then
+			local Vec=Pos-obj:GetPos()
+			local Dist,Dir=Vec:Length(),Vec:GetNormalized()
+			if SERVER and Dist/MaxRange < 0.1 then
```
Comment (Jackarunda) [inline lua/entities/ent_jack_gmod_ezblackhole.lua]: i'm not sure about this, i'd like the damage to scale based on distance/lifetime but i'll deal with that after the merge

File: lua/entities/ent_jack_gmod_ezblackhole.lua (hunk 1 above)
Comment (TheOnly8Z) [reply lua/entities/ent_jack_gmod_ezblackhole.lua]: Good catch, though it shouldn't affect anything. Amended.

File: lua/entities/ent_jack_gmod_ezblackhole.lua (hunk 2 above)
Comment (TheOnly8Z) [reply lua/entities/ent_jack_gmod_ezblackhole.lua]: I'll submit a commit with a simple scaling. Might not work too well considering breakables have different health values, but I don't believe it's possible to scale based off prop max hp.

**Pull request merged**